### DISCUSSION TITLE
Posibility to upgrade leading edge slat textures

### DISCUSSION THREAD

**Discussion by scotgr** [flybywiresim/aircraft#3053] [Ideas]
I would like to put forward a suggestion to update the leading edge slat textures to look more realistic as currently they are not corrrect and appear almost chrome and mirror like.

Also you guys are doing a fantastic job and I love the acheviements you are making.  Keep up the great work !

Thanks
Scott :)

**Comment by hotshotp** [comment]
thats rather a material than a texture issue, we cant edit that before we get out blender exporter working, but when we do we will definitely take a look

**Reply by scotgr** [reply]
Hi hotshotp, thanks for the reply.  hats great that you will have a look at this :)  Its a small thing but does make a difference!

**Reply by hotshotp** [reply]
Thank you! Yeah this is absolutely a thing we are looking at, there are a lot of inaccuracies that we need to fix :)

**Reply by scotgr** [reply]
I cant wait to see this project develop :)
